docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_while_exploring_data_in_a_notebook_without_using_a_data_context.rst

```diff
@@ -3,16 +3,227 @@
+Steps
+-----
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+All of these steps take place within your notebook:
+
+1. **Import Great Expectations.**
+
+    .. code-block:: python
+
+        import great_expectations as ge
+
+2. **Load some data.**
+
+    The simplest way to do this is with ``read_csv``.
+
+    .. code-block:: python
+
+        my_df = ge.read_csv("my_data_directory/titanic.csv")
+
+    This method behaves exactly the same as ``pandas.read_csv``, so you can add parameters to parse your file:
+    
+    .. code-block:: python
+
+        my_df = ge.read_csv(
+            "my_data_directory/my_messy_data.csv",
+            sep="\t",
+            skiprows=3
+        )
+
+    Similarly wrapped versions of other pandas methods (``read_excel``, ``read_table``, ``read_parquet``, ``read_pickle``, ``read_json``, etc.) are also available. Please see the <great_expectations utils module> for details.
+
+    If you wish to load data from somewhere else (e.g. from a SQL database or blob store), please fetch a copy of the data locally, or :ref:`configure a Datasource <tutorials__getting_started__connect_to_data>`. (In the future, you will likely want to configure Datasources anyway, to take advantage of more of Great Expectations' advanced features.)
+
+    As another option, if you have already instantiated a ``pandas.Dataframe``, you can use ``from_pandas``:
+
+    .. code-block:: python
+
+        my_df = ge.from_pandas(
+            my_pandas_dataframe
+        )
+
+    This method will convert your boring old pandas ``DataFrame`` into a new and exciting great_expectations ``PandasDataAsset``. The two classes are absolutely identical, except that ``PandasDataAsset`` has access to Great Expectations' methods.
+
+3. **Explore your data and add Expectations.**
+
+    Each of the methods in step 1 will produce ``my_df``, a ``PandasDataAsset``. ``PandasDataAsset`` is a subclass of ``pandas.DataFrame``, which means that you can use all of pandas' normal methods on it.
+
+    .. code-block:: python
+
+        my_df.head()
+        my_df.Sex.value_counts()
+        my_df[my_df.Sex=="male"].head()
+        # etc., etc. 
+        
+    In addition, ``my_df`` has access to a wide array of Expectations. You can see the full list :ref:`here <Glossary of Expectations>`. By convention, every Expectation method name starts with the name ``expect_...``, so you can quickly access the full list with tab-based autocomplete:
+
+    .. image:: ../../images/expectation_autocomplete.gif
+
+    |
+
+    When you invoke an Expectation, it will immediately be validated against your data. The returned object will contain the result and a list of exceptions. This instant feedback helps you zero in on exceptions very quickly, taking a lot of the guesswork out of data exploration.
```

Comment:
```suggestion
    When you invoke an Expectation, it will immediately be validated against your data. The returned object will contain the result and a list of unexpected values. This instant feedback helps you zero in on unexpected data very quickly, taking a lot of the guesswork out of data exploration.
```

Reply:
ooh. Good catch.